### DISCUSSION TITLE
Setting OTEL_TRACES_EXPORTER=none does not stop the trace exporter

### DISCUSSION THREAD

**Discussion by kwmlodozeniec** [open-telemetry/opentelemetry-java-instrumentation#6936] [Q&A] [answered]
In my environment I set OTEL_TRACES_EXPORTER to none during local development but I still get repeated exceptions in the logs as traces are being exported. Any ideas how to disable the export completely?

My stack uses FastAPI and Requests instrumentation and the exception I'm getting is

```
ConnectionError(e, request=request)\nrequests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=4318): Max retries exceeded with url: /v1/traces
```

**Comment by mateuszrzeszutek** [comment, marked as answer]
Hey @kwmlodozeniec ,
You should probably open this discussion in https://github.com/open-telemetry/opentelemetry-python-contrib - FastAPI and Requests are not Java libs

**Reply by kwmlodozeniec** [reply]
sorry, got the repos confused, too many tabs open.

**Reply by mateuszrzeszutek** [reply]
No prob!
